<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HatsuClass</class>
 <widget class="QMainWindow" name="HatsuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>456</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>456</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>彩-深红</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Hatsu/Resources/icon.ico</normalon>
    <disabledoff>:/Hatsu/Resources/icon32.ico</disabledoff>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>设定</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="postprocess">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>压制参数</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>21</y>
            <width>171</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>CRF</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="crfBox">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>37.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
               <property name="value">
                <double>26.399999999999999</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Encoder</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="x264Selector"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="IsCareColor">
             <property name="text">
              <string>在乎色彩</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="IsCareMontion">
             <property name="text">
              <string>在乎动态</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="IsCareTime">
             <property name="text">
              <string>在乎时间</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="preprocess">
         <property name="title">
          <string>视频前处理</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>141</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="preProcess_None">
             <property name="text">
              <string>无</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="preProcess_Gaussian">
             <property name="text">
              <string>高斯模糊</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="preProcess_Bilateral">
             <property name="text">
              <string>双边模糊</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="preProcess_SurfaceBlur">
             <property name="text">
              <string>表面模糊</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="preProcess_BM3D">
             <property name="text">
              <string>BM3D</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="videoFile">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="videoSelector">
           <property name="text">
            <string>视频</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="audioFile">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="audioSelector">
           <property name="text">
            <string>音频</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Scripting">
     <attribute name="title">
      <string>脚本</string>
     </attribute>
     <widget class="QPlainTextEdit" name="ScriptEditor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QTextEdit" name="About">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>321</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="commandCompress">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>压制</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>videoFile</tabstop>
  <tabstop>audioFile</tabstop>
  <tabstop>videoSelector</tabstop>
  <tabstop>audioSelector</tabstop>
  <tabstop>crfBox</tabstop>
  <tabstop>commandCompress</tabstop>
  <tabstop>IsCareColor</tabstop>
  <tabstop>IsCareMontion</tabstop>
  <tabstop>IsCareTime</tabstop>
  <tabstop>x264Selector</tabstop>
  <tabstop>preProcess_None</tabstop>
  <tabstop>preProcess_Gaussian</tabstop>
  <tabstop>preProcess_Bilateral</tabstop>
  <tabstop>preProcess_SurfaceBlur</tabstop>
  <tabstop>preProcess_BM3D</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>ScriptEditor</tabstop>
 </tabstops>
 <resources>
  <include location="hatsu.qrc"/>
 </resources>
 <connections/>
</ui>
